src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    public void notEnoughAge_Exception_OK() {
```

Comment:
```suggestion
    public void register_notEnoughAge_NotOK() {
```
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/c0f68ceb-52ef-47fe-8e75-98328bf76875)